drivers/display/Kconfig.st7789v

```diff
@@ -9,6 +9,25 @@ menuconfig ST7789V
 	help
 	  Enable driver for ST7789V display driver.
 
+choice ST7789V_SERIAL_INTERFACE
+	prompt "Serial interface"
+	default ST7789V_SERIAL_4_LINE
+	depends on ST7789V
+	help
+	  Specify the serial interface for the ST7789V display controller.
+	  The 3-lines serial data packet contains a control bit D/CX and
+	  a transmission byte. In 4-lines serial interface, data packet
+	  contains just transmission byte and control bit D/CX is transferred
+	  by the D/CX pin.
+
+config ST7789V_SERIAL_3_LINE
+	bool "3-line"
+
+config ST7789V_SERIAL_4_LINE
+	bool "4-line"
+
+endchoice
```

Comment:
this forces all instances to use one mode or the other, can this be moved to Devicetree instead?

Reply:
I thought so too, but the same is true for the pixel format, and I wanted to keep the changes to a minimum. A follow-up PR?

Reply:
You would keep changes minimal if it used devicetree.

Reply:
I've removed the `Kconfig` option and reworked the logic to use an optional cmd/data pin. If it is not set, it operates in the 3-line serial protocol.